yaki_mobile/lib/presentation/features/declaration/declaration_half_day_end.dart

```diff
@@ -101,7 +101,7 @@ class DeclarationHalfDayEnd extends ConsumerWidget {
                             declarationMode:
                                 DeclarationPaths.fromText(declarationMode),
                             teamList: teamList,
-                            buttonValue: StatusEnum.onSite,
+                            buttonValue: StatusEnum.remote,
```

Comment:
StatusEnum.onSite.name, not StatusEnum.office.name

Reply:
it's remote and not onSite. remote existing already on file status page utils.  it's concerning remote. there was an error with onSite